### PR TITLE
Cleanup pep8 warnings

### PR DESCRIPTION
**Pull request by gyorb**
This is a new pull request because there were many refactorings since #419 was started.

### CODE REVIEW

**Comment by Xazax-hun** [comment]
This pull request is very big. Maybe we should split it into two separate one for tests and one for the lib?

**Comment by gyorb** [comment]
Because this pull request contains mostly formatting I do not think it is hard to review. I planed to put the missing modules here too but if you think this will get too big I will create separate pull requests for them. I wanted to do this as fast as possible, because if we start a new feature there can be a lot of conflicts which is not always easy to resolve especially if there is some refactoring too.

File: codechecker_lib/checkers.py [hunk 1]
```diff
@@ -13,7 +13,8 @@ def get_env_checkers_list(env_name):
     env_set_checkers = util.get_env_var(env_name)
 
     LOG = logger.get_new_logger("CHECKERS")
-    LOG.debug_analyzer('Getting checkers list from environment variable %s' % (env_name))
+    LOG.debug_analyzer('Getting checkers list from environment variable %s'
+                       % (env_name))
```
Comment (Xazax-hun) [inline codechecker_lib/checkers.py]: Redundant paren around env_name

File: codechecker_lib/plist_helper.py [hunk 2]
```diff
@@ -72,42 +72,103 @@ def get_check_name(current_msg):
...
-    "Undefined or garbage value returned to caller": "core.uninitialized.UndefReturn",
-    "Use of memory after it is freed": "alpha.unix.MallocWithAnnotations",
-    "Value stored to  during its initialization is never read": "deadcode.DeadStores",
-    "Value stored to  is never read": "deadcode.DeadStores"
-}
+    {
+        "": "NOT FOUND",
+
+        "Access out-of-bound array element (buffer overflow)":
+
```
Comment (Xazax-hun) [inline codechecker_lib/plist_helper.py]: Is this a good idea to have this much new lines? Does this make it more readable? Maybe exporting this to a JSON is better?

Comment (gyorb) [inline codechecker_lib/plist_helper.py]: Good idea but I would do this as a separate issue. Many things could be refactored but just focus on pep8 cleanup now.

File: codechecker_lib/checkers.py (hunk 1 above)
Comment (gyorb) [inline codechecker_lib/checkers.py]: Done.

**Pull request merged**